src/WorkBC.Importers.Federal/Program.cs

```diff
@@ -95,5 +111,45 @@ private static async Task Main(string[] args)
             //Done
             logger.Information("IMPORTER FINISHED");
         }
+
+        /// <summary>
+        ///     Create persistent HttpClient
+        /// </summary>
+        private static HttpClient CreateHttpClient(IConfiguration configuration)
+        {
+            //get Proxy settings
+            var proxySettings = new ProxySettings();
+            configuration.GetSection("ProxySettings").Bind(proxySettings);
+            
+            //get API settings
+            var federalSettings = new FederalSettings();
+            configuration.GetSection("FederalSettings").Bind(federalSettings);
+            
+            //create persistent HttpClient
+            var handler = new HttpClientHandler();
+
+            if (proxySettings.UseProxy)
+            {
+                handler.Proxy = new WebProxy(proxySettings.ProxyHost, proxySettings.ProxyPort)
+                {
+                    BypassProxyOnLocal = true
+                };
+            }
+
+            if (proxySettings.IgnoreSslErrors)
+            {
+                handler.ClientCertificateOptions = ClientCertificateOption.Manual;
+                handler.ServerCertificateCustomValidationCallback =
+                    (httpRequestMessage, cert, cetChain, policyErrors) => true;
+            }
+
+            handler.AutomaticDecompression = DecompressionMethods.Deflate | DecompressionMethods.GZip;
+            var httpClient = new HttpClient(handler);
+            httpClient.Timeout = new TimeSpan(0,0,5);
+            httpClient.DefaultRequestHeaders.Add("Cookie", federalSettings.AuthCookie);
+
+            return httpClient;
```

Comment:
Instead of creating an `HttpClient` on every request, we create it once and reuse it again and again.  This significantly improves performance and makes our API service easier to test since the `HttpClient` that we pass in can be mocked.  See associated tests.